src/playground/models/user.clj

```diff
@@ -0,0 +1,12 @@
+(ns playground.models.user
+  (:require [clojure.spec.alpha :as spec]
+            [clojure.string :as string]))
+
+(spec/def ::username (spec/and string? seq (complement clojure.string/blank?)))
```

Comment:
Do this:

`(spec/def ::present-string (spec/and string? seq (complement clojure.string/blank?)))`

Then:

`(spec/def ::name ::present-string)`. Same for the other 2 string fields